### PR TITLE
[MFTF] Cover adding Category image via WYSIWYG  from Enhanced Media Gallery

### LINKED ISSUE #1073: [MFTF] Cover adding Category image via WYSIWYG from Enhanced Media Gallery
### Preconditions (*)
1. Installed Magento with Adobe Stock Integration
2. Configured  integration in Stores -> Configuration -> System -> Adobe Stock Integration fieldset

### Steps to reproduce (*)
Admin user is logged in to admin panel
Go to Stores - Configuration under Settings section 
Assert that Configuration window is opened
User expands Advanced tab
User click System
User expands Enhanced Media Gallery tab
User selects Yes from Enabled dropdown
User clicks Save config button
Assert that the configuration has been saved 
Navigate to cms/page page
Click Add New Page button 
Expand Content fieldset
Click Show/Hide Editor button
Click Insert Image ... button
Click Search Adobe Stock button
User expands image preview
User click Save Preview 
Click Confirm button
Image appears in Media Gallery
Navigate to Catalog > Categories
User select Root Category form category tree
User clicks Add Subcategory button
User types Category name
User expands Content tab
User click Insert/edit image under Description section
Assert that the Insert/edit image window appeared
User click Search source
Assert that the Enhanced Media Gallery opened
User select image from Enhanced Media Gallery
User clicks Ass Selected button
Assert that the link appeared in Source field
User click OK
Assert that the image is displayed in Description Editor
User Saves category
Open Storefront
Go to the previously created Category

### Expected result (*)
Assert that category image is displayed

### Note (*)
Related issue - https://github.com/magento/adobe-stock-integration/issues/1068
Test on Cucumber Studio - https://studio.cucumber.io/projects/131313/test-plan/folders/943908/scenarios/4484351

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#1073: Cover adding Category image via WYSIWYG from Enhanced Media Gallery

### Manual testing scenarios (*)
All tests green :heavy_check_mark:

### CODE REVIEW

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

File: MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryAddCategoryImageTest.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <actionGroup ref="AdminDisableWYSIWYGActionGroup" stepKey="disableWYSIWYG" />
+            <deleteData createDataKey="category" stepKey="deleteCategory"/>
+        </after>
+        <actionGroup ref="AdminOpenCategoryPageActionGroup" stepKey="openCategoryPage"/>
+        <actionGroup ref="AdminCategoriesOpenCategoryActionGroup" stepKey="openCategory">
+            <argument name="category" value="$$category$$"/>
+        </actionGroup>
+        <actionGroup ref="AdminOpenMediaGalleryTinyMce4ActionGroup" stepKey="openMediaGalleryFromWysiwyg"/>
+        <actionGroup ref="AdminEnhancedMediaGallerySearchAdobeStockActionGroup" stepKey="openAdobeStockGrid"/>
+        <actionGroup ref="AdminAdobeStockImagePreviewConditionalCloseActionGroup" stepKey="closePreviewIfOpened"/>
```
Comment (sivaschenko) [inline MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryAddCategoryImageTest.xml]: The condition click was moved to AdminAdobeStockExpandImagePreviewActionGroup - AdminAdobeStockImagePreviewConditionalCloseActionGroup action group can be removed

File: AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockImagePreviewConditionalCloseActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (sivaschenko) [inline AdobeStockImageAdminUi/Test/Mftf/ActionGroup/AdminAdobeStockImagePreviewConditionalCloseActionGroup.xml]: The condition click was moved to AdminAdobeStockExpandImagePreviewActionGroup - AdminAdobeStockImagePreviewConditionalCloseActionGroup action group can be removed

File: MediaGalleryUi/Test/Mftf/Section/AdminMediaGalleryHeaderButtonsSection.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="AdminMediaGalleryHeaderButtonsSection">
+        <element name="addSelected" type="button" selector=".media-gallery-add-selected"/>
+        <element name="imageInGrid" type="button" selector="//img[contains(@src, '{{imageName}}')]" parameterized="true"/>
```
Comment (sivaschenko) [inline MediaGalleryUi/Test/Mftf/Section/AdminMediaGalleryHeaderButtonsSection.xml]: Please move this element to AdminEnhancedMediaGalleryImageActionsSection

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @Nazar65 ! There are few minor changes required

**Review by gabrieldagama — APPROVED** [review]

**Comment by Nazar65** [comment]
@magento run all tests

**Pull request merged**